packages/replay-next/pages/variables.css

```diff
@@ -119,6 +119,16 @@
   --z-index-7--mask: var(--z-index-7);
   --z-index-8--modal: var(--z-index-8);
   --z-index-8--dropdown: var(--z-index-8);
+
+  /* The styles below are the same between themes */
+  /* TODO [FE-2246] Are there other light-only styles like this that we should colocate? */
+  --support-form-color: var(--grey-90);
+  --support-form-close-icon-color: #ffffff;
+  --support-form-modal-background-color: #ffffff;
+  --support-form-modal-border-color: var(--grey-90);
+  --support-form-link-background-color: var(--grey-20);
+  --support-form-link-background-color-hover: var(--grey-30);
+  --support-form-text-area-background-color: var(--grey-20);
```

Comment:
I don't _love_ this but I don't want to hard-code these values deep inside of the app either.